.github/workflows/mdns__host-tests.yml

```diff
@@ -0,0 +1,61 @@
+name: "mdns: host-tests"
```

Comment:
here although we building host and AFL tests in one file, will be nice to show in title

Reply:
Same here. Naming mentioned below https://github.com/espressif/esp-protocols/pull/242/files#diff-76592f40f1f57d7e25dc47a9c8e919d9801e57f323dad7fbcc5dce5845331093R37
(since these are not AFL tests, just checkers that we're still able to compile when we want to run the AFL tests)